package.json

```diff
@@ -19,6 +19,8 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.5.4",
     "@babel/register": "^7.4.4",
+    "@vue/test-utils": "^1.0.0-beta.29",
```

Comment:
It seems to me that you moved all dev dependencies to the top level, so I did the same. Am I right ?

Reply:
Yes you're right